api-lib/event_triggers/activity/index.ts

```diff
@@ -45,7 +45,7 @@ export default async function handler(req: VercelRequest, res: VercelResponse) {
           event: {
             op: operation,
             data: {
-              new: { id, circle_id, created_at },
+              new: { id, circle_id, created_at, created_by },
```

Comment:
wondering if created_by should be created_by_profile or something since we have a lot of ambiguity about profile vs user in the app